manifests/params.pp

```diff
@@ -33,6 +33,35 @@
         ],
         'maxconn' => '8000'
       }
+        $config_file      = '/etc/haproxy/haproxy.cfg'
+      }
+    'FreeBSD': {
+      $package_name     = 'haproxy'
+      $global_options   = {
+        'log'           => [
+          '127.0.0.1 local0',
+          '127.0.0.1 local1 notice',
+        ],
+        'chroot'        => '/usr/share/haproxy',
```

Comment:
not the log location.  i meant the log destinations and facilities that are above, but i think they're good.  we need to set a destination, because (from the haproxy man page):
```
Since HAProxy can run inside a chroot, it cannot reliably access /dev/log. For this reason, it uses the UDP protocol to send its logs to the server, even if it is the local server.
```